make/dev.mk

```diff
@@ -0,0 +1,106 @@
+ifndef DEV_MK
+DEV_MK:=# Prevent repeated "-include".
+
+include ./make/verbose.mk
+include ./make/git.mk
+
+DOCKER_REPO?=quay.io/openshiftio
+IMAGE_NAME?=tektoncd-pipeline-operator
+REGISTRY_URI=quay.io
+
+DEVCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/tektoncd-pipeline-operator
+TIMESTAMP:=$(shell date +%s)
+TAG?=$(GIT_COMMIT_ID_SHORT)-$(TIMESTAMP)
+OPENSHIFT_VERSION?=4
+
+# to watch all namespaces, keep namespace empty
+APP_NAMESPACE ?= ""
+LOCAL_TEST_NAMESPACE ?= "local-test"
+
+.PHONY: local
+## Run Operator locally
+local: deploy-rbac build deploy-crd
+	$(Q)-oc new-project $(LOCAL_TEST_NAMESPACE)
+	$(Q)operator-sdk up local --namespace=$(APP_NAMESPACE)
+
+.PHONY: deploy-rbac
+## Setup service account and deploy RBAC
+deploy-rbac:
+	$(Q)-oc login -u system:admin
+	$(Q)-oc create -f deploy/service_account.yaml
+	$(Q)-oc create -f deploy/role.yaml
+	$(Q)-oc create -f deploy/role_binding.yaml
+
+.PHONY: deploy-crd
+## Deploy CRD
+deploy-crd:
+	$(Q)-oc apply -f deploy/crds/devconsole_v1alpha1_component_crd.yaml
```

Comment:
those are not the crds that operator owns

Reply:
Sorry I will swamp that out for the correct ones for tekton